src/App/App.tsx

```diff
@@ -1,32 +1,30 @@
 import React, { useState, useEffect } from 'react'
-import { Route } from 'react-router-dom'
 import './App.scss'
 import Header from '../Header/Header'
 import View from '../View/View'
-import { getCoordinates } from '../helpers/apiCalls'
 
 const App: React.FC = () => {
-	// const [ topCities, setTopCities ] = useState([])
+	const [ topCities, setTopCities ] = useState([])
 
-	useEffect(() => {getTop5Cities()}, [])
+	// useEffect(() => {getTop5Cities()}, [])
```

Comment:
Lines 9-18 can get deleted if y'all are cool with it?

Reply:
set them free. Code is cheap.